src/lib/util/state.ts

```diff
@@ -162,9 +162,13 @@ export const toggleDarkTheme = (dark: boolean): void => {
 	});
 };
 
+let urlDebounce: number;
 export const initURLSubscription = (): void => {
 	stateStore.subscribe(({ serialized }) => {
-		history.replaceState(undefined, undefined, `#${serialized}`);
+		clearInterval(urlDebounce);
```

Comment:
`clearInterval` does the same thing as `clearTimeout`, but it's a bit confusing if you're actually clearing a timeout.

```suggestion
		clearTimeout(urlDebounce);
```